src/en/help-rackspace.md

```diff
@@ -51,6 +71,12 @@ Which will ask for a credential name, and the authentication type. Enter
 need to set the default one to use with `juju set-default-credential`.
 
 
+### 3. Create and use a `~/.novarc` file
+
+Place the AWS information in a '~/.novarc' or similarly named file
+containing the environment variables and assigned values. See the official
+[OpenStack](http://docs.openstack.org/icehouse/install-guide/install/apt/content/cli_openrc.html) documentation.
+
```

Comment:
I don't know. I do see a glaring bit of incorrectness, though. 'AWS' instead of 'Rackspace/OpenStack' in line 76. I'll fix that and research the rest.